src/php/ext/grpc/channel.c

```diff
@@ -572,6 +651,8 @@ GRPC_STARTUP_FUNCTION(channel) {
   gpr_mu_init(&global_persistent_list_mu);
   le_plink = zend_register_list_destructors_ex(
       NULL, php_grpc_channel_plink_dtor, "Persistent Channel", module_number);
+  zend_hash_init_ex(&grpc_persistent_list, 20, NULL,
```

Comment:
Maybe a constant for 20?

Reply:
I think it's just initiate size. It can resize dynamically.
When I set it to 2, the list still can be added to ten or more.

Reply:
I was just referring [other people's code](https://github.com/php/php-src/blob/a7fe2570d3ce6915d4ea85c62c0f880ddc225ba7/Zend/zend_list.c#L220), choose a random number. 20 sound's like a good number for the max number of persistent channels which I am going to set as default.